app/src/androidTest/java/com/greenfox/peridot/peridot_coz_android/ExampleInstrumentedTest.java

```diff
@@ -22,5 +22,11 @@ public void useAppContext() throws Exception {
         Context appContext = InstrumentationRegistry.getTargetContext();
 
         assertEquals("com.greenfox.peridot.peridot_coz_android", appContext.getPackageName());
+
     }
+
+
+
+
+
```

Comment:
first we wanted to write the test here, so we were moving the block of lines here and there, it stayed there by accident